client/src/components/AllTabs/AllTabs.jsx

```diff
@@ -35,19 +30,19 @@ function AllTabs({dataLogs}) {
         {activeTab === allTabs[0] ? (
           <VolumeGraph dataLogs={dataLogs}/>
         ) : activeTab === allTabs[1] ? (
-          <PainGraph dataLogs = {dataLogs}/>
+          <TemplateGraph dataLogs = {dataLogs} type = {"pain"}/>
         ) : activeTab === allTabs[2] ? (
-          <BowelsGraph dataLogs = {dataLogs}/>
+            <TemplateGraph dataLogs = {dataLogs} type = {"bowels"}/>
         ) : activeTab === allTabs[3] ? (
-          <BreathingGraph dataLogs = {dataLogs}/>
+            <TemplateGraph dataLogs = {dataLogs} type = {"breathing"}/>
         ) : activeTab === allTabs[4] ? (
-          <AppetiteGraph dataLogs = {dataLogs}/>
+            <TemplateGraph dataLogs = {dataLogs} type = {"appetite"}/>
         ) : activeTab === allTabs[5] ? (
-          <NauseaGraph dataLogs = {dataLogs}/>
+            <TemplateGraph dataLogs = {dataLogs} type = {"nausea"}/>
         ) : activeTab === allTabs[6] ? (
-          <FatigueGraph dataLogs = {dataLogs}/>
+            <TemplateGraph dataLogs = {dataLogs} type = {"fatigue"}/>
         ) : activeTab === allTabs[7] ? (
-          <SleepingGraph dataLogs = {dataLogs} />
+            <TemplateGraph dataLogs = {dataLogs} type = {"sleeping"}/>
```

Comment:
Do we need all these if-else statements?

Why not do `<TemplateGraph dataLogs = {dataLogs} type = {allTabs[i]}/>` in the else case?

Reply:
Oh this is true yes I can change this!